src/Components/Calculator.js

```diff
@@ -1,97 +1,87 @@
-            4
-          </button>
-          <button type="button" className="col-3 text-center border p-3" onClick={this.btnClick}>
-            5
-          </button>
-          <button type="button" className="col-3 text-center border p-3" onClick={this.btnClick}>
-            6
-          </button>
-          <button type="button" className="col-3 text-center border p-3 bg-warning" onClick={this.btnClick}>
-            -
-          </button>
-        </div>
-        <div className="row">
-          <button type="button" className="col-3 text-center border p-3" onClick={this.btnClick}>
-            1
-          </button>
-          <button type="button" className="col-3 text-center border p-3" onClick={this.btnClick}>
-            2
-          </button>
-          <button type="button" className="col-3 text-center border p-3" onClick={this.btnClick}>
-            3
-          </button>
-          <button type="button" className="col-3 text-center border p-3 bg-warning" onClick={this.btnClick}>
-            +
-          </button>
-        </div>
-        <div className="row">
-          <button type="button" className="col-6 text-center border p-3" onClick={this.btnClick}>
-            0
-          </button>
-          <button type="button" className="col-3 text-center border p-3" onClick={this.btnClick}>
-            .
-          </button>
-          <button type="button" className="col-3 text-center border p-3 bg-warning" onClick={this.btnClick}>
-            =
-          </button>
-        </div>
+  const btnClick = (e) => {
+    setCalAction((obj) => calculate(obj, e.target.innerText));
+  };
+  const { total, operation, next } = calAction;
+  return (
+    <div>
+      <div className="row">
+        <div className="col border p-4 bg-secondary text-light ">{(total || '') + (operation || '') + (next || '') || 0 }</div>
       </div>
-    );
-  }
-}
+      <div className="row">
+        <button type="button" className="col-3 text-center border p-3" onClick={btnClick}>
+          AC
+        </button>
+        <button type="button" className="col-3 text-center border p-3" onClick={btnClick}>
+          +/-
+        </button>
+        <button type="button" className="col-3 text-center border p-3" onClick={btnClick}>
+          %
+        </button>
+        <button type="button" className="col-3 text-center border p-3 bg-warning" onClick={btnClick}>
+          &divide;
+        </button>
+      </div>
+      <div className="row">
+        <button type="button" className="col-3 text-center border p-3" onClick={btnClick}>
+          7
+        </button>
+        <button type="button" className="col-3 text-center border p-3" onClick={btnClick}>
+          8
+        </button>
+        <button type="button" className="col-3 text-center border p-3" onClick={btnClick}>
+          9
+        </button>
+        <button type="button" className="col-3 text-center border p-3 bg-warning" onClick={btnClick}>
+          x
```

Comment:
- [OPTIONAL] 🤔 This could have been a separate component (`Buttons`) in a separate file and rendered here. Smaller components are easy to fix, reusable, and make the app more scalable. :pray: 👍🏻